src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -484,6 +508,61 @@ async def _genomic_to_transcript_exon_coordinate(
 
         params = {key: None for key in TranscriptExonData.model_fields}
 
+        if get_nearest_transcript_junction:
+            # Check if is_fusion_transcript_segment is given
```

Comment:
This uses the old param name. I don't think it's a useful since you're just stating check the value of the param.

```suggestion
```